src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaFlowRepositoryTest.java

```diff
@@ -155,6 +199,7 @@ public void shouldUpdateFlow() {
         flow.setBandwidth(10000);
         flow.setEncapsulationType(FlowEncapsulationType.TRANSIT_VLAN);
 
+        assertTrue(flowRepository.findById(TEST_FLOW_ID).isPresent());
```

Comment:
I added this and the similar asserts just by the way to reduce inspection warnings about using get without checking if there is something in the Optional.